### PR TITLE
Move login button

### PR DESCRIPTION
**Pull request by skimaharvey**

### CODE REVIEW

File: components/LoginBtn/LoginBtn.tsx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+  }, [walletAddress]);
+
+  const renderLogin = () => (
+    <>
+      <Link href="/connect-wallets">
+        <button className="flex items-center rounded-lg text-deepPink bg-white py-1 px-4">
+          <span className="mx-2 text-sm">Connect</span>
+        </button>
+      </Link>
+    </>
```
Comment (Hugoo) [inline components/LoginBtn/LoginBtn.tsx:129]:
```suggestion
      <Link href="/connect-wallets">
        <button className="flex items-center rounded-lg text-deepPink bg-white py-1 px-4">
          <span className="mx-2 text-sm">Connect</span>
        </button>
      </Link>
```

File: components/LoginBtn/LoginBtn.tsx [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+    ethereum.on('chainChanged', function (networkId: number) {
+      if (networkId !== L16_CHAIN_ID) {
+        setWalletAddress('');
+        toast('Please switch to L16 network');
+      }
+    });
+  };
+
+  const addressFirstFour = () => walletAddress?.slice(2, 6);
```
Comment (Hugoo) [inline components/LoginBtn/LoginBtn.tsx:105]: why make it a function?

Why not just

```suggestion
  const addressFirstFour = walletAddress?.slice(2, 6);
```

File: components/LoginBtn/LoginBtn.tsx [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+import { ERC725, ERC725JSONSchema } from '@erc725/erc725.js';
+import erc725Schema from '@erc725/erc725.js/schemas/LSP3UniversalProfileMetadata.json';
+import identicon from 'ethereum-blockies-base64';
+
+import { WalletAddressContext } from '../../contexts/WalletAddressContext';
+import useWalletConnect from '../../hooks/useWalletConnect';
+import { IPFS_GATEWAY_BASE_URL, L16_CHAIN_ID } from '../../constants';
+import { toast } from 'react-toastify';
+import walletConnectConnector from '../../utils/walletConnectConnector';
+import Link from 'next/link';
```
Comment (Hugoo) [inline components/LoginBtn/LoginBtn.tsx:11]: I suggest to put external libs at the top

File: components/LoginBtn/LoginBtn.tsx [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+      if (LSP3ProfileImage.includes('ipfs://')) {
+        return `${IPFS_GATEWAY_BASE_URL}${
+          LSP3ProfileImage.split('ipfs://')[1]
+        }`;
+      }
+
+      return LSP3ProfileJSON?.LSP3Profile?.profileImage[0].url;
+    }
+    return identicon(walletAddress);
+  };
```
Comment (Hugoo) [inline components/LoginBtn/LoginBtn.tsx:45]: i think this should go in its own component - but lets keep it here for now

File: components/LoginBtn/LoginBtn.tsx [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+    const erc725 = new ERC725(
+      erc725Schema as ERC725JSONSchema[],
+      walletAddress,
+      web3Provider,
+      config,
+    );
+    const LSP3Profile = await erc725.fetchData(['LSP3Profile']);
+    const formattedLSP3 = validateLSP3(LSP3Profile[0].value);
+    setLsp3JSON(formattedLSP3);
+  };
```
Comment (Hugoo) [inline components/LoginBtn/LoginBtn.tsx:71]: i think these utils function should be put outside of the component, as they can be used elsewhere. Not just here. You could import this function here

**Review by Hugoo — CHANGES_REQUESTED** [review]
Could you also put a screenshot when you do UI changes? It helps to see things. Otherwise we have to clone and run locally to see the changes.

**Comment by skimaharvey** [comment]
Screenshot 2022-09-21 at 17 43 22

**Review by Hugoo — APPROVED** [review]

**Pull request merged**